.dotnet/tests/TestScenarios/Batch/BatchTests.cs

```diff
@@ -1,21 +1,49 @@
 ﻿using NUnit.Framework;
 using OpenAI.Batch;
 using OpenAI.Files;
+using OpenAI.Tests.Utility;
 using System;
 using System.ClientModel;
 using System.IO;
+using System.Text.Json;
 using System.Threading.Tasks;
 using static OpenAI.Tests.TestHelpers;
 
 namespace OpenAI.Tests.Batch;
 
-public partial class BatchTests
+[TestFixture(true)]
+[TestFixture(false)]
+public partial class BatchTests : SyncAsyncTestBase
 {
+    public BatchTests(bool isAsync)
+        : base(isAsync)
+    {
+    }
+
     [Test]
     public async Task ListBatchesProtocol()
     {
         BatchClient client = GetTestClient();
-        ClientResult result = await client.GetBatchesAsync(after: null, limit: null, options: null);
+        ClientResult result = IsAsync
+            ? await client.GetBatchesAsync(after: null, limit: null, options: null)
+            : client.GetBatches(after: null, limit: null, options: null);
+
+        BinaryData response = result.GetRawResponse().Content;
+        JsonDocument jsonDocument = JsonDocument.Parse(response);
+        JsonElement dataElement = jsonDocument.RootElement.GetProperty("data");
+
+        Assert.That(dataElement.GetArrayLength(), Is.GreaterThan(0));
+
+        long unixTime2024 = (new DateTimeOffset(2024, 01, 01, 0, 0, 0, TimeSpan.Zero)).ToUnixTimeSeconds();
```

Comment:
I put an `s_2024` in AssistantTests that we could reuse for this (e.g. via pulling into the TestBase) if it's helpful -- got sick of writing all those zeroes, too!